src/main/scala/carldata/series/Patterns.scala

```diff
@@ -0,0 +1,19 @@
+package carldata.series
+
+import java.time.temporal.ChronoUnit
+import java.time.{Instant, LocalDateTime, ZoneOffset}
+
+object Patterns {
+  def daily[V](xs: TimeSeries[V])(implicit num: Fractional[V]): TimeSeries[V] = {
+    def groupBy5Minutes(dt: Instant): Instant = {
```

Comment:
It would be better to infer pattern resolution instead of assuming 5 minutes

Reply:
Sorry my mistake.
It is not necessary to infer resolution here.
Just group by time (rest date). This will be enough